app/views/content/how-to-apply-for-teacher-training/when-to-apply-for-teacher-training.md

```diff
@@ -0,0 +1,60 @@
+---
+title: When to apply for teacher training
+description: |-
+  Find out when you can apply for postgraduate teacher training courses, deadlines for submitting applications and how the interview process works.
+related_content:
+    
+navigation: 30.45
+navigation_title: When to apply for teacher training
+navigation_description:  Find out when you can apply for postgraduate teacher training courses.
+date: "2023-03-30"
+keywords:
+  - teacher training applications
+---
+
+You can apply for postgraduate teacher training courses starting in September 2024 from 10 October 2023 to 17 September 2024. 
+
+## Find and apply for teacher training courses
+
+From 10 October 2023, you can <a href="https://www.gov.uk/apply-for-teacher-training">apply for a teacher training course</a>.
+ 
+You can apply to up to 4 courses at a time.  
+
+Training providers should respond to your application within 30 days. If you do not receive a decision from your application after 30 days, you can submit an application for another course. 
+
+It’s wise to apply as soon as you’re ready because: 
+<ul>
+<li> popular courses will fill up quickly </li>
+<li>there are a limited number of places on a course and they are filled up on a first come, first served basis</li>  
+<li>courses are open for people to apply all year round</li>  
+<li>you may need time to complete a <a href="https://getintoteaching.education.gov.uk/how-to-apply-for-teacher-training/subject-knowledge-enhancement">subject knowledge enhancement (SKE)</a> course and other conditions before starting your training</li> 
+<li>this will give you time to plan how you <a href="https://getintoteaching.education.gov.uk/funding-and-support ">fund your training</a></li>
+</ul>
+
+“Applying late was stressful. I was limited in my choice of teacher training course provider, and I'd lost my GCSE certificates. I had to go back to my school and get new copies so I could confirm my place on the course and my student finance.” 
+
+If you are applying outside of England, you can <a href="https://getintoteaching.education.gov.uk/teacher-training-adviser/sign_up/identity">sign up for a teacher training adviser</a> to get help with the application process. 
```

Comment:
```suggestion
You can <a href="/teacher-training-adviser/sign_up/identity">sign up for a teacher training adviser</a> to get help with the application process. 
```
Internal links don't need the domain :-)

Also I know this was in the original draft but I don't think we should imply that advisers are only for non-UK candidates